.travis.yml

```diff
@@ -28,10 +28,11 @@ branches:
 
 matrix:
   include:
-    - php: 5.5
-      env:
-        - COMPOSER_FLAGS="--prefer-lowest"
-        - SYMFONY_VERSION='2.8.*'
+# fails to install. try to get this to work again before releasing
+#    - php: 5.5
+#      env:
+#        - COMPOSER_FLAGS="--prefer-lowest"
+#        - SYMFONY_VERSION='2.8.*'
```

Comment:
jordi thinks we hit some hard to tell edge case: https://twitter.com/seldaek/status/805698883456270336

i comment this build out for now and try again when we released the library